### ISSUE TITLE
run limited set of rules / exclude rules

### ISSUE THREAD

**Issue by wopian** [actionshub/markdownlint#7]
## 😦  Problem Statement

(GitHub doesn't have `:person_frowning:` as a shortcode for 🙍)

Right now this action runs mdl with all the lint rules active. 

I maintain a project that has multilingual README/CONTRIBUTING files that contain right-to-left languages, which require HTML tags along with assembly code examples that conflict with spacing rules. As it stands there is no way to disable these lint rules from being run.

## :grey_question: Possible Solution

Matching mdl's behaviour:
- If rules are defined, only those rules are run (e.g `mdl --rules MD001` only checks `MD001`)
- If rules are prefixed with `~`, those rules are excluded (`mdl --rules ~MD001` checks every rule except `MD001`)

The following configuration would call `mdl --rules ~MD001,~MD002`:

```yaml
jobs:
  delivery:
    runs-on: ubuntu-latest
    steps:
    - name: Check out code
      uses: actions/checkout@master
    - name: Run mdl
      uses: actionshub/markdownlint@master
      with:
        rules: ~MD001,~MD002
```

## :heavy_plus_sign: Additional context

I currently have a fork with the rules I need disabled hardcoded in the entrypoint.sh at https://github.com/wopian/markdownlint

**Comment by Xorima** [comment]
I guess I have a question as to why not use an `.mdlrc` file to configure this instead, https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md#mdl-configuration

Is this something that would cover your usecase? I personally use it over: https://github.com/sous-chefs/vscode/blob/master/.mdlrc, as it works both in github and local runs of `mdl`

**Comment by wopian** [comment]
I did not realise `mdl` had a config file - I searched GitHub Actions for markdown linters and this was the first maintained action I found. I did look up the ruby gem, but I guess I found the `--rules` CLI command before the configuration file 😅 

It may be worth mentioning the `mdl` configuration using `.mdlrc` in the README however.

**Comment by Xorima** [comment]
Glad I could help,

Do you want to issue a pr to add the information into the readme?

Thanks
Jason

**Comment by Xorima** [comment]
So I have added the docs as I needed to do a release anyway. Thanks for this and the docker image fix, released in tag 1.2.1 (though pointing at master would work as well)
